css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+html {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 100px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+p {
+  font-size: 30px;
+  color: rgb(134, 134, 134);
+  line-height: 60px;
+  width: 80%;
+}
+
+h2 {
+  font-size: 20px;
+  width: 30%;
+}
+
+.caption {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: 300px;
+  font-size: 30px;
+  text-transform: uppercase;
+  font-weight: lighter;
+  width: auto;
+}
+
+.head {
+  display: flex;
+  font-size: 30px;
+  text-align: center;
+  color: rgb(134, 134, 134);
+}
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  padding: 10px;
+}
+
+.column-eighth {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: calc(100% / 8);
+  align-items: flex-start;
+}
+
+.column-full {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+}
+
+.column-two-eighth {
```

Comment:
column classes should also have a width set on them, otherwise you are letting flexbox control the size of the elements in this example, and we don't want that!